Makefile

```diff
@@ -38,6 +42,14 @@ check:
 		sort | uniq > /tmp/compat-links
 	@ (diff /tmp/compat-defs /tmp/compat-defs)
 
+install: $(addprefix install-,compat-el)
+
+install-compat-el: $(BYTEC) $(BYTEC:.elc=.el)
```

Comment:
I am not familiar with this pattern could you explain what you are trying to achieve here?

Reply:
The pattern replaces the .elc in BYTEC with .el so the .el files are also installed.

Reply:
> I wasn't referring to substitution references, rather the need to declare a `install-compat-el` target, instead of just handling everything in `install`.

Because the install-compat-el targets makes use of the substitution rule below which iterates over each dependency.